jsonpb/jsonpb_test.go

```diff
@@ -721,6 +721,61 @@ func TestUnmarshalingBadInput(t *testing.T) {
 	}
 }
 
+type funcResolver func(turl string) (proto.Message, error)
+
+func (fn funcResolver) Resolve(turl string) (proto.Message, error) {
+	return fn(turl)
+}
+
+func TestAnyWithCustomResolver(t *testing.T) {
+	resolveCount := 0
+	resolver := funcResolver(func(turl string) (proto.Message, error) {
```

Comment:
Minor.  Capture turl and verify that it got the expected TypeUrl value.

Reply:
Done